fastqfil/additional_scripts/filtration_fastq.py

```diff
@@ -0,0 +1,14 @@
+# Checking the length, GC content and reading quality of nucleotide chains
+def filt_fastq(name, sequence, quality, gc_bounds, length_bounds, quality_threshold):

```

Comment:
Поскольку функция не выполняет фильтрацию, а проверяет подходит ли цепочка под переданные условия и возвращает булево значение - более инфомативно было бы назвать ее не `filt_fastq`.

Сходу подобрать название сложновато, но вот к примеру:
```suggestion
def check_sequence_conditions(name, sequence, quality, gc_bounds, length_bounds, quality_threshold):
```
или 
```suggestion
def is_sequence_valid(name, sequence, quality, gc_bounds, length_bounds, quality_threshold):
```